packages/web3torrent/config/webpack.config.js

```diff
@@ -75,7 +75,7 @@ module.exports = function(webpackEnv) {
         [
           'VERSION',
           'CHAIN_NETWORK_ID',
-          'DRIFT_CHATBOX_ID',
+          'DRIFT_CHATBOX_APP_ID',
```

Comment:
Do we still want to whitelist environment variables? As is we need to add a new env var in two places (`.env.something` as well as here). Easy to miss this, and it doesn't match the pattern used in our other apps.

Reply:
I don't think it's that useful to be whitelisting them

Reply:
https://github.com/statechannels/monorepo/pull/2045